FAQ.md

```diff
@@ -87,29 +89,33 @@ This is set via the MCO’s config, not the Machine-api. The config a Machine re
 ## How does the user-data get created for the VMs?
 The user-data is generated by the installer.  The Machine-api is just a consumer of the user-data and simply passes the data to the cloud.  Refer to documentation for the Machine-config-operator (not part of the Machine-api) for creating new Machine pools and user-data.
 
-## Adding Labels to Nodes via Machines
-Labels are only added to Nodes via Machines at creation time.  You can add arbitrary labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent labels on the Machine object (or in a MachineSet’s template) will have no effect.
+## Adding Annotations and Labels to Nodes via Machines
+Annotations and Labels are only added to Nodes via Machines at creation time.  You can add arbitrary annotations and labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent annotations and labels on the Machine object (or in a MachineSet’s template) will have no effect.
 
-### Which labels get added to a Node?
-When we talk about Node labels here, it can be somewhat confusing.
+### Which Annotations and Labels get added to Nodes?
+MachineSets can be used to help mark the Machines and Nodes that are created from them by using Annotations and Labels. To do this you will need to add this information in various places on your MachineSet depending on where you would like this information. Please note that this information will only be applied to new Machines and Nodes created from the MachineSet, they will not be retroactively applied.
 
-To add labels to a Node on a MachineSet, the place where you should add labels is **.spec.template.spec.metadata.labels.**
+Setting labels or annotations in `MachineSet.Spec.Template.Metadata` will cause them to be applied to every Machine object created from that MachineSet. Specifically these labels and annotations will end up in `Machine.Metadata`.
 
-To add labels to a Node on a Machine that's not part of a MachineSet (and only
-    upon first creating the Machine object): **.spec.metadata.labels.**
+Setting labels or annotations in the `MachineSet.Spec.Template.Spec.Metadata` will cause them to be applied to every Machine and Node object created from the MachineSet. In the case of Machine objects, these labels and annotations will be applied in `Machine.Spec.Metadata`. For Node objects, these values will be applied to `Node.Metadata`.
```

Comment:
I wonder if `applied to every machine` implies they would be on the top level metadata rather than in the spec? What is the connotation of a label being applied to something?

Reply:
i wasn't trying to imply any deeper meaning, just trying to reflect what the user needs to set for something to end up in the related resource. i'm happy to reword this, maybe just say "added" instead of "applied"?

Reply:
It's fine as is, don't worry about changing